pkgs/development/haskell-modules/with-packages-wrapper.nix

```diff
@@ -2,6 +2,7 @@
 , withLLVM ? false
 , postBuild ? ""
 , ghcLibdir ? null # only used by ghcjs, when resolving plugins
+, ghcOptions ? []
```

Comment:
Could you add a comment for this line, ideally with an example of what can be passed here?  I think an example like `["-j16" "+RTS" "-A64M" "-RTS"]` makes it immediately obvious to Haskellers what these options are.